### PR TITLE
Added documentation on how to renew ssl certs

### PR DESCRIPTION
**Pull request by joaquinrz**
# Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
To add instructions on how to renew ssl certificates

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/1094

### CODE REVIEW

File: docs/renewCertificates.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+SecretValue=$(cat certificate.pfx | base64)
+
+export KEYVAULT_NAME=<keyvault-name>
+
+az keyvault secret set --vault-name $KEYVAULT_NAME --name sslcert --value ${SecretValue}
+
+export APP_GATEWAY_RG_NAME=<app-gateway-rg-name>
+export APP_GATEWAY_NAME=<app-gateway-name>
+export KEYVAULT_SECRET_ID=<kv-secret-id> # must be full format e.g. https://kv-aks-xxx.vault.azure.net/secrets/sslcertAustinRdc/xyz123
```
Comment (AAkindele) [inline docs/renewCertificates.md:34]: Recommend fetching the secret id here so the user won't have to look it up.
```suggestion
export KEYVAULT_SECRET_ID=$(az keyvault secret set --vault-name $KEYVAULT_NAME --name sslcert --value ${SecretValue} --query id -o tsv)

export APP_GATEWAY_RG_NAME=<app-gateway-rg-name>
export APP_GATEWAY_NAME=<app-gateway-name>
```

File: docs/renewCertificates.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+az keyvault secret set --vault-name $KEYVAULT_NAME --name "appgw-ingress-internal-aks-ingress-tls" --file "ngsa_bundle.crt"
+
+az keyvault secret set --vault-name $KEYVAULT_NAME --name "appgw-ingress-internal-aks-ingress-key" --file "_.austinrdc.dev.key"
+```
+
+## Renew Codespaces secrets
+
+Last step is to refresh the following codespace secrets with the contents its respective file
+
+```text
```
Comment (AAkindele) [inline docs/renewCertificates.md:55]: Should the contents of these files be base64 encoded before adding to Codespace secrets?

**Review by AAkindele — APPROVED** [review]
👍

File: docs/renewCertificates.md (hunk 2 above)
Comment (joaquinrz) [reply docs/renewCertificates.md:54]: great catch, thanks @AAkindele

**Pull request merged**